### DISCUSSION TITLE
Controller Pak not recognised

### DISCUSSION THREAD

**Discussion by elpardo84** [darthcloud/BlueRetro#747] [Q&A]
BlueRetro firmware version

[1.8.3] 2023-02-15 have also tried with 1.8.4 but then factory reset back to 1.8.3

BlueRetro firmware specification

HW1

BlueRetro firmware variant

System specific

BlueRetro hardware type

External adapter dongle (1 port only)

Manufacturer

8bitmods

System used

Nintendo 64

Bluetooth controller brand & name

Nintendo Switch Online N64 controller

What is problem? (only list ONE problem per report)

I cant get the controller pak to work. Every game i try it said "Controller pak not found” or “controller pak may be damaged try to repair".

I’ve tried multiple games, factory reset, formatted the memory pak via the web gui. I’m not using any hdmi adapter, I’ve tried the n64 with and without the expansion pak and using the composite cable and rf adapter.

What did you expect to happen?

to use the memory card to save

**Comment by darthcloud** [comment]
Are you using a N64 Digital or Ultra HDMI mods?

**Reply by elpardo84** [reply]
No completely unmodded

**Reply by c-m-green** [reply]
I am experiencing this problem as well. NSO64 controller on original console. Have tried updating to 1.9.1 before resetting back to 1.8.4.

**Reply by darthcloud** [reply]
@c-m-green Are you saying 1.8.4 got not cpak issue while v1.9.1 does? Or they both behave the same?

**Reply by c-m-green** [reply]
@darthcloud They both behave the same way.
